hudi-common/src/main/java/org/apache/hudi/common/bloom/filter/BloomFilterFactory.java

```diff
@@ -52,6 +52,7 @@ public static BloomFilter createBloomFilter(int numEntries, double errorRate, in
    * @return the {@link BloomFilter} thus generated from the passed in serialized string
    */
   public static BloomFilter fromString(String serString, String bloomFilterTypeCode) {
+
```

Comment:
can we remove the additional line break?

Reply:
Done.